### PR TITLE
Ronnie/thread name pathname

### PR DESCRIPTION
**Pull request by ronnie-stratoscale**
Thread name is shown as an optional column, while pathname:lineno are added to the end of the message

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 .PHONY: test run-example run-example-dynamic client
 
 test:
-	go test -race ./... -timeout 30s
+	go test  ./... -timeout 30s
```
Comment (posener) [inline Makefile]: please restore the `-race` flag

Comment (ronnie-stratoscale) [reply Makefile]: oops

**Comment by posener** [comment]
Closes #18

**Comment by posener** [comment]
@ronnie-stratoscale I've seen that the path is not visible in the UI, right?

**Pull request merged**